sesamutils/sesamlogger.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import os
+
+
+def sesam_logger(logger_name, timestamp=False):
+    logger = logging.getLogger(logger_name)
+    level = None
+    if os.getenv('LOG_LEVEL') is not None:
+        level = logging.getLevelName(os.getenv('LOG_LEVEL').upper())
+    stdout_handler = logging.StreamHandler()
+    if timestamp:
+        stdout_handler.setFormatter(logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s'))
```

Comment:
If you use the if condition to to define the string used by the formatter, you wouldn't have to write out `stdout_handler.setFormatter(logging.Formatter(` twice. This also helps keeping the lines shorter.